portal/src/graphql/portal/ManageLanguageWidget.tsx

```diff
@@ -387,8 +435,15 @@ const ManageLanguageWidget: React.VFC<ManageLanguageWidgetProps> =
 
     const onRenderOption: IRenderFunction<IDropdownOption> = useCallback(
       (option?: IDropdownOption) => {
+        const style: CSSProperties | undefined = option?.disabled
+          ? {
+              fontStyle: "italic",
+              color: theme.semanticColors.disabledText,
+            }
+          : undefined;
+
         return (
-          <Text>
+          <Text style={style}>
```

Comment:
`style` is the ordinary style attribute of a HTML element. Fluent UI supports `styles`. `styles` can be a function. You can receive `theme` from the arguments. Then you can save calling `useTheme()` manually.